frontend/src/metabase/admin/datamodel/components/filters/pickers/DatePicker/RangeDatePicker.tsx

```diff
@@ -98,14 +98,15 @@ export const BetweenPicker = ({
           onClear={handleEndDateClear}
         />
       </DateContainer>
-      <div className="Calendar--noContext">
+      <div>
         <Calendar
           isRangePicker
           primaryColor={primaryColor}
           initial={endValue}
           selected={startValue && moment(startValue)}
           selectedEnd={endValue && moment(endValue)}
           onChangeDate={handleDateClick}
+          noContext={true}
```

Comment:
Instead of having the `:global(.Calendar--noContext .Calendar-day--this-month)` class, I just use a prop here to keep the CSS classes within one component.